includes/api.php

```diff
@@ -277,6 +277,10 @@ public static function send_feedback( $feedback_key, $feedback_text ) {
 	public static function ajax_reset_api_data() {
 		check_ajax_referer( 'elementor_reset_library', '_nonce' );
 
+		if ( ! current_user_can( 'manage_options' ) ) {
+			wp_send_json_error( 'Permission denied' );
+		}
```

Comment:
is only admin has access to the library?

Reply:
it's the reset library button under admin tools